index.html

```diff
@@ -49,13 +52,17 @@
     <body>
         <header>
             <h1>Offentligt IT</h1>
-            <h2>Offentlige midler, offentlig software</h2>
+            <h2>Offentlige midler, offentlig IT</h2>
         </header>
         <section class="intro">
             <p>
-                <strong>Offentlig IT</strong> har til formål at forbedre offentlig IT gennem
-                oplysning og debat. Ved at sætte fokus på åbenhed og uafhængighed, viser vi 
-                vejen til en billigere og bedre digital infrastruktur, til gavn for hele Danmark. 
+                <strong>Offentlig IT</strong> er et projekt, der har til
+                formål at åbne og forbedre vores offentlige IT infrastruktur - til gavn for hele Danmark.
```

Comment:
Kan godt lide "åbne og forbedre"!

Der skal vist være bindestreg i IT-infrastruktur.

Jeg synes ikke vi behøver "til gavn for hele Danmark".